zebrad/Cargo.toml

```diff
@@ -60,7 +60,7 @@ zebra-test = { path = "../zebra-test" }
 
 [features]
 enable-sentry = []
-test_sync_to_canopy_mainnet = []
-test_sync_to_canopy_testnet = []
-test_sync_past_canopy_mainnet = []
-test_sync_past_canopy_testnet = []
+test_sync_to_mandatory_checkpoint_mainnet = []
+test_sync_to_mandatory_checkpoint_testnet = []
+test_sync_past_mandatory_checkpoint_mainnet = []
+test_sync_past_mandatory_checkpoint_testnet = []
```

Comment:
This change is ok, but it also needs an update to the GitHub workflows:
https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/regenerate-stateful-test-disks.yml#L68
https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/regenerate-stateful-test-disks.yml#L69
https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/test.yml#L72

If you're using `fastmod`, it skips hidden files and directories like `.github` by default.
You can use `fastmod --hidden` to change them as well, but you'll need `fastmod` version 0.4.2.